I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 142, in script
    define y = Character(name)
  File "game/script.rpy", line 142, in <module>
    define y = Character(name)
NameError: name 'name' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 142, in script
    define y = Character(name)
  File "/Users/emilyamspoker/renpy-8.2.1-sdk/renpy/ast.py", line 2219, in execute
    self.set()
  File "/Users/emilyamspoker/renpy-8.2.1-sdk/renpy/ast.py", line 2233, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "/Users/emilyamspoker/renpy-8.2.1-sdk/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 142, in <module>
    define y = Character(name)
NameError: name 'name' is not defined

macOS-12.3-arm64-arm-64bit arm64
Ren'Py 8.2.1.24030407
Paws and Play 1.0
Tue Apr  9 17:46:35 2024
